convert_to_dhtmlx.py

```diff
@@ -57,7 +57,7 @@ def convert_ical_event(self, calendar_event):
             "url": calendar_event.get("URL"),
             "id": (uid, start_date),
             "type": "event",
-            "color": calendar_event.get("X-APPLE-CALENDAR-COLOR", "")
+            "color": calendar_event.get("COLOR", "") || calendar_event.get("X-APPLE-CALENDAR-COLOR", "")
```

Comment:
I would not use the || that is JavaScript - does that exist in Python? The value would be put into the default argument. I guess, if I merge this, it is a syntax error... did you check that?

Reply:
The idea is good but this is invalid syntax.

`calendar_event.get("COLOR", calendar_event.get("X-APPLE-CALENDAR-COLOR", ""))` ?

Reply:
Yes, I would say that works.

Reply:
Ahh oops, that is 100% my fault! I'm a bit overworked right now and I just bashed out the fix when I saw the comment on #52. Sorry again; good spot.

Reply:
Should be fixed now :)